server/src/termdb.violin.js

```diff
@@ -8,7 +8,7 @@ import { write_file } from './utils'
 import { getData } from './termdb.matrix'
 import { createCanvas } from 'canvas'
 import { violinBinsObj } from '../../server/shared/violin.bins'
-import { summaryStats } from '../../server/shared/descriptive.stats'
+import summaryStats from '../../server/shared/descriptive.stats'
```

Comment:
path should be `../shared/descriptive.stats`

Reply:
thanks. should be fixed now. Will get rid of the sleep in Barchart integration in the next PR. Doesn't seem to be working with detect handlers.